lib/vty.h

```diff
@@ -338,6 +339,12 @@ struct vty_arg {
 /* Vty read buffer size. */
 #define VTY_READ_BUFSIZ 512
 
+/* Vty max send buffer size (4096 * 4096). */
+#define SEND_BUF_MAX 16777216
+
+/* Vty flush intermediate size (32 * 4096). */
+#define MAX_INTERMEDIATE_FLUSH 131072
+
```

Comment:
if you want to put these in the public vty.h header, please follow the naming pattern and call them `VTY_xxx`

Reply:
Will do